hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILECrck.py

```diff
@@ -84,15 +91,15 @@ def cache_dir(self) -> Path:
 
     def setup(self) -> None:
         if self.encoder_type == SSLEncoder.__name__:
-            from InnerEye.ML.configs.histo_configs.run_ids import innereye_ssl_checkpoint_crck_4ws
+            from histopathology.configs.run_ids import innereye_ssl_checkpoint_crck_4ws
             self.downloader = CheckpointDownloader(
-                aml_workspace=get_workspace(),
+                azure_config_json_path=get_workspace(),
                 run_id=innereye_ssl_checkpoint_crck_4ws,
                 checkpoint_filename="best_checkpoint.ckpt",
                 download_dir="outputs/",
                 remote_checkpoint_dir=Path("outputs/checkpoints")
             )
-            os.chdir(fixed_paths.repository_root_directory().parent)
+            os.chdir(fixed_paths.repository_parent_directory())
```

Comment:
repository_parent_directory doesn't exist in fixed_paths, instead this should be repository_root_directory().parent (see previous version)